mitodl/mit-open
# Remove learning_format, use delivery for search filter/facet

### What are the relevant tickets?
Part 2 of https://github.com/mitodl/hq/issues/5126, depends on https://github.com/mitodl/course-search-utils/pull/123

### Description (What does it do?)
- Uses an updated version of course-search-utils (https://github.com/mitodl/course-search-utils/pull/123), ~~which needs to be merged before this one.  Once it is merged and released, then this PR will be updated to use the new release version.  Currently it is pinned to the course-search-utils PR commit~~.
- Removes the `learning_format` field from the `LearningResource`  model
- Renames `learning_format` to `delivery` in the `Profile` model
- Removes "learning_format" from the ETL pipelines 
- Updates the frontend to use "delivery" instead of "learning_format" for filtering and facets



### How can this be tested?
- Run some searches, with and without using the "Format" facet, everything should work as expected.
- Update your preferred format for your user profile, everything should work as expected.

### Additional Context
